### PR TITLE
Running L3AFD in a Docker container

### PR DESCRIPTION
**Pull request by aka320**
Changes
---------
- Added Dockerfile to build the Docker image for L3AFD
- Update README with the instructions to build L3AFD Docker image and run it in a Container

Testing
-------
- Able to attach XDP and TC programs using L3AFD Container

### CODE REVIEW

**Review by pmoroney — APPROVED** [review]
LGTM, although it does sound like a good idea to pin the hash for the docker file as suggested by the security bot.

File: README.md [hunk 1]
```diff
@@ -60,6 +60,35 @@ go test -tags WINDOWS ./...
 # Generate Swagger Docs
 See our [Swaggo setup](docs/swagger.md)
 
+# Build L3AFD Docker image
```
Comment (sanfern) [inline README.md:63]: Move this block in building section

File: build-docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    zlib1g-dev \
+    libevent-dev \
+            vim \
+            wget \
+            curl \
+            linux-tools-generic \
+            net-tools \
+            iproute2 \
+            elfutils \
+            libjson-c-dev \
```
Comment (sanfern) [inline build-docker/Dockerfile:26]: Fix alignment.

**Review by sanfern — CHANGES_REQUESTED** [review]

**Review by sanfern — APPROVED** [review]
LGTM

**Review by charleskbliu0 — APPROVED** [review]
Looks good

**Pull request merged**